### ISSUE TITLE
info/error/warning messages to the metrics sent to prometheus

### ISSUE THREAD

**Issue by hmsvigle** [derailed/popeye#125]
<img src="https://raw.githubusercontent.com/derailed/popeye/master/assets/popeye_feature.png" align="right" width="100" height="auto"/>

<br/>
<br/>
<br/>

We are getting below metrics from popeye to prometheus.

![Prometheus-Metrics](https://user-images.githubusercontent.com/24938159/91994918-c9638600-ed54-11ea-9e5b-b67e095099a6.png)

There is no "message" field in these reports to extract. 
So that, we can try to fetch these messages separately along with the count, which will server the purpose of having the metrics generated. 

![popeye-sanitizer_report](https://user-images.githubusercontent.com/24938159/91994888-bfda1e00-ed54-11ea-8565-6b4b3a47fdfe.png)

**Is your feature request related to a problem? Please describe.**
Basically these reports, when populated to prometheus might not server the purpose to troubleshooting. Hence is it feasible to add error/warning/Info messages to the filter for all the metrics generated ?

**Describe the solution you'd like**
1. My request is to add error/info/warning Messages to the metrics.
2. It would be helpful, if above messages could be mapped resource-wise/level-wise. 

**Describe alternatives you've considered**
I have thought of parsing the json output of popeye to implement this requirement. alternatively looking for other tool, what would provide next step of easy integration to prometheus & grafana. 

**Additional context**
Please let me know, if any further clarifications required.

**Comment by hmsvigle** [comment]
@eminugurkenar @derailed Kindly look into this feature request. 

I have created dashboard in grafana as per data sent to prometheus through pushgateway. Currently, I am able to display only the total no of Error/Warn/Ok/Info Messages for every sanitizer. To get the detailed message, I would have to parse the report & segregate type of messages.  

It would be beneficial if we can add info/Error/Warn/Ok Messages in the filter itself of existing Metrices or by creating separate Metrics. So that I would be able to display the message itself in Grafana dashboard, instead of parsing the report to segregate Messages. 

In this way, it will be a complete solution. 
1. Generate the sanitation report
2. Visualize it in Grafana - Along with the counts(numbers), all Messages as well to take appropriate action on them.

Thanks

**Comment by derailed** [comment]
Fixed v0.20.0
